### PR TITLE
feat: Added StoreCollectionClient class useful for listing Actors in Apify Store

### PR DESCRIPTION
**Pull request by drobnikj**
New method for listing Actors in store
See API reference https://docs.apify.com/api/v2/#/reference/store

### CODE REVIEW

**Review by fnesveda — APPROVED** [review]
Looks good, and works well!

File: src/resource_clients/store_collection.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import ow from 'ow';
+import { ApiClientSubResourceOptions } from '../base/api_client';
+import { ResourceCollectionClient } from '../base/resource_collection_client';
+import { PaginatedList } from '../utils';
+import { ActorStats } from './actor';
+
+/**
+ * @hideconstructor
```
Comment (B4nan) [inline src/resource_clients/store_collection.ts:8]: FYI this has no effect on the generated docs, you need to put a `@hidden` jsdoc over the constructor itself (this problem is present in many other places in this repo)

File: src/resource_clients/store_collection.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+     * https://docs.apify.com/api/v2/#/reference/store/store-actors-collection/get-list-of-actors-in-store
+     * @param {object} [options]
+     * @param {number} [options.limit]
+     * @param {number} [options.offset]
+     * @param {string} [options.search]
+     * @param {string} [options.sortBy]
+     * @param {string} [options.category]
+     * @param {string} [options.username]
+     * @param {string} [options.pricingModel]
+     * @return {Promise<PaginationList>}
```
Comment (B4nan) [inline src/resource_clients/store_collection.ts:28]: i'd remove this, it adds no value, the information is all available on type level (and it wont have any effect on the docs either)

**Review by B4nan — APPROVED** [review]
just two nits

**Pull request merged**